GitUI/CommitInfo/CommitInfo.Designer.cs

```diff
@@ -250,6 +251,6 @@ private void InitializeComponent()
         private CommitInfoHeader commitInfoHeader;
         private System.Windows.Forms.Panel pnlCommitMessage;
         private System.Windows.Forms.RichTextBox rtbxCommitMessage;
-        private GitUI.UserControls.DoubleBufferedTableLayoutPanel tableLayout;
+        private System.Windows.Forms.TableLayoutPanel tableLayout;
```

Comment:
If you don't think `DoubleBufferedTableLayoutPanel` is useful, then we should remove it completely.
I added it whilst working on the commit info rendering.

Reply:
ok